javascript/README.md

```diff
@@ -1,3 +1,5 @@
 # @ruby/prism
 
 JavaScript bindings for Ruby's [prism](https://github.com/ruby/prism) parser.
+
+See the [JavaScript documentation](../docs/javascript.md) for Prism.
```

Comment:
This isn't going to get shipped with the package, so the link is going to be broken for npm/node_modules. So maybe link absolutely? Like https://github.com/ruby/prism/blob/main/docs/javascript.md maybe?